src/App.jsx

```diff
@@ -13,34 +13,83 @@ export const goodsFromServer = [
   'Jam',
   'Garlic',
 ];
+const SORT_FIELD_ALPHABETICCALY = 'alphabetically';
```

Comment:
There is a typo in the constant name `SORT_FIELD_ALPHABETICCALY`. It should be `SORT_FIELD_ALPHABETICALLY` to correctly reflect the intended meaning.